data-serving/data-service/src/geocoding/adminsFetcher.ts

```diff
@@ -83,6 +83,16 @@ export default class MapboxAdminsFetcher {
                         (await this.getName(feature.properties.id)) ||
                         undefined;
             }
+            // Necessary because the schema doesn't accept undefined values
```

Comment:
we should change the schema to allow undefined values for those, leave a todo if you don't want to do it here?

Reply:
Added todo.